pkg/ansible/paramconv/paramconv.go

```diff
@@ -70,6 +71,42 @@ func ToCamel(s string) string {
 	return ret
 }
 
+// This function modifies the string to handle special words that are a part of wordMapping
```

Comment:
This one is not very readable. I am not sure if could not have an easy way/more understandable way to achieve this goal. However, could we improve the comments here to make clear that it is used to not change/snake the special worlds in the wordMapping? Could we add in the comment an example over the scenario and the expected result?